cypress-tests/cypress/e2e/PaymentTest/00015-ZeroAuthMandate.cy.js

```diff
@@ -122,39 +142,38 @@ describe("Card - SingleUse Mandates flow test", () => {
       let data = getConnectorDetails(globalState.get("connectorId"))["card_pm"][
         "ZeroAuthPaymentIntent"
       ];
-      let req_data = data["Request"];
-      let res_data = data["Response"];
+
       cy.createPaymentIntentTest(
         fixtures.createPaymentBody,
-        req_data,
-        res_data,
+        data,
         "no_three_ds",
         "automatic",
         globalState
       );
+
       if (should_continue)
-        should_continue = utils.should_continue_further(res_data);
+        should_continue = utils.should_continue_further(data);
     });
 
     it("Confirm No 3DS payment", () => {
       let data = getConnectorDetails(globalState.get("connectorId"))["card_pm"][
         "ZeroAuthConfirmPayment"
       ];
-      let req_data = data["Request"];
-      let res_data = data["Response"];
-      cy.confirmCallTest(
-        fixtures.confirmBody,
-        req_data,
-        res_data,
-        true,
-        globalState
-      );
+
+      cy.confirmCallTest(fixtures.confirmBody, data, true, globalState);
+
       if (should_continue)
-        should_continue = utils.should_continue_further(res_data);
+        should_continue = utils.should_continue_further(data);
     });
 
     it("Retrieve Payment Call Test", () => {
-      cy.retrievePaymentCallTest(globalState);
+      let data = getConnectorDetails(globalState.get("connectorId"))["card_pm"][
+        "ZeroAuthConfirmPayment"
+      ];
+
+      let configs = validateConfig(data["Configs"]);
+
+      cy.retrievePaymentCallTest(globalState, configs);
```

Comment:
```suggestion
      let data = getConnectorDetails(globalState.get("connectorId"))["card_pm"][
        "ZeroAuthConfirmPayment"
      ];
      cy.retrievePaymentCallTest(globalState, data);
```
here also pass only the `data` to  retrievePaymentCallTest, you are already doing validateConfig inside

Change it in other places also

Reply:
oh, i guess i missed it here. will get it done and update the flow in the attached issue.